<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCG_loader</class>
 <widget class="QMainWindow" name="PCG_loader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(0, 0, 0);
font-family: arial, verdana;
}

QPushButton{
background-color:rgb(0, 0, 0);
color:rgb(176, 176, 0);
font-family: arial;
font-size: 14px;
}

QLabel{
color:rgb(176, 176, 0);
font-family: arial;
font-size: 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="GraphicsLayoutWidget" name="plt1">
      <property name="styleSheet">
       <string notr="true">GraphicsLayoutWidget{
background-color: rgb(0, 0, 0);

}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="GraphicsLayoutWidget" name="plt2">
      <property name="styleSheet">
       <string notr="true">GraphicsLayoutWidget{
background-color: rgb(0, 0, 0);

}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="buttonLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="stopButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-color: rgb(0, 85, 255);
font-size: 38px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="openButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-color: rgb(0, 85, 255);
font-size: 38px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="playButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-color: rgb(0, 85, 255);
font-size: 38px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QPushButton" name="labelButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-color: rgb(0, 85, 255);
font-size: 38px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>Set Label</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">QLabel{
border:0;
font-size: 18px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Label Value&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="label_value_sb"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="saveLabelButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-color: rgb(0, 85, 255);
font-size: 38px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>Save Labels</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="loadLabelButton">
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-style: outset;
border-width: 1px;
border-color: rgb(0, 85, 255);
font-size: 38px;
background-color: back;
color: rgb(231, 243, 0);
font-family:arial;
}</string>
        </property>
        <property name="text">
         <string>Load Labels</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
